scripts/test-acceptance.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
```

Comment:
Not a must, but a brief description of what this script does (and what it does with args) would be nice.